documentation/book/proc-resizing-persistent-volumes.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// assembly-storage.adoc
+
+[id='proc-resizing-persistent-volumes-{context}']
+= Resizing persistent volumes
+
+This procedure describes how to increase the size of persistent volumes used by existing {ProductName} cluster.
+This procedure can be applied to clusters using a single persistent volumes as well as to clusters using multiple persistent volumes in a JBOD storage.
+
+NOTE: {ProductPlatformName} currently supports only increasing size of persistent volumes.
+Shrinking of volumes is currently not possible.
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster with support for volume resizing
```

Comment:
I thought the platforms we refer to supported resizing full stop? When don't they?

Reply:
The volume resizing depends on a lot of things:
* Version of Kubernetes / OpenShift
* The configuration of Kubernetes / OpenShift (It is supported in some versions on different support levels and can be enabled / disabled by the admins)
* Storage infrastructure which supports volume resizing

So it can fairly easily happen that it will not be supported. But at the same time it is quite hard to determine the support for us, so we cannot be more specific.